pkg/proxy/handlers/spacelister.go

```diff
@@ -49,11 +52,13 @@ func (s *SpaceLister) HandleSpaceListRequest(ctx echo.Context) error {
 		if len(workspaces) != 1 {
 			// not found
 			r := schema.GroupResource{Group: "toolchain.dev.openshift.com", Resource: "workspaces"}
+			metrics.RegServProxyResponseHistogramVec.WithLabelValues(metrics.ResponseMetricLabelRejected).Observe(time.Since(startTime).Seconds())
 			return errorResponse(ctx, apierrors.NewNotFound(r, workspaceName))
 		}
+		metrics.RegServProxyResponseHistogramVec.WithLabelValues(metrics.ResponseMetricLabelRejected).Observe(time.Since(startTime).Seconds())
```

Comment:
Shouldn't this be `metrics.ResponseMetricLabelRouted` instead of rejected since in the next line it's calling getWorkspaceResponse()?

Reply:
yes, thanks for catching that!